### PR TITLE
add routes for getting user, pass to component and fetch user for ver…

### PR DESCRIPTION
**Pull request by JMStudiosJoe**
…ification and connect to login

### CODE REVIEW

File: backend/routes/userRoutes.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }).catch(error => {
+        console.log(error)
+        return res.status(522).json(error)
+    })
+})
+
+usersRouter.get('/api/user/:id', (req, res) => {
+    db.getById('users', req.params.id)
+    .then(user => {
+        delete user.passphrase
```
Comment (clarle) [inline backend/routes/userRoutes.js]: I don't love the fact that the user object in memory has the passphrase available.  I feel like someone could very easily accidentally log the user's passphrase.  

Can we fetch all of the user fields from Mongo, minus the passphrase? 

https://stackoverflow.com/questions/12096262/how-to-protect-the-password-field-in-mongoose-mongodb-so-it-wont-return-in-a-qu

File: backend/routes/userRoutes.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express'
+import { db } from '../lib/db'
+import { users_db_name } from '../Utilities/API_utilities'
+
+
+const usersRouter = Router()
+
+
+usersRouter.post('/api/user', (req, res) => {
```
Comment (clarle) [inline backend/routes/userRoutes.js]: Do we want only admins to be able to create new users?

File: src/containers/AdminContainer/AdminContainer.js [hunk 3]
```diff
@@ -19,6 +18,50 @@ class AdminContainer extends React.Component {
...
+            ...state,
+            userID: userID,
+        }
+        return newState
+    }
+
+    componentDidMount() {
+        getUser(this.state.userID)
+        .then(response => {
+            if (response['error']) {
```
Comment (clarle) [inline src/containers/AdminContainer/AdminContainer.js]: In `getUser`, if there's an `error` property in `response`, it should throw so you can handle the error in the catch instead.

File: backend/routes/userRoutes.js (hunk 2 above)
Comment (JMStudiosJoe) [reply backend/routes/userRoutes.js]: Yes but need to lock down routes in another PR

File: backend/routes/userRoutes.js (hunk 1 above)
Comment (JMStudiosJoe) [reply backend/routes/userRoutes.js]: Yes lets make an issue for this along with locking down the routes

File: src/containers/AdminContainer/AdminContainer.js (hunk 3 above)
Comment (JMStudiosJoe) [reply src/containers/AdminContainer/AdminContainer.js]: there shouldnt be anymore since fixing the backend will not send a successful error

**Pull request merged**